src/Compilers/CSharp/Portable/Binder/Imports.cs

```diff
@@ -160,7 +160,8 @@ internal sealed class Imports
                             continue;
                         }
 
-                        var imported = usingsBinder.BindNamespaceOrTypeSymbol(usingDirective.Name, diagnostics, basesBeingResolved);
+                        var declarationBinder = usingsBinder.WithAdditionalFlags(BinderFlags.SuppressConstraintChecks);
+                        var imported = declarationBinder.BindNamespaceOrTypeSymbol(usingDirective.Name, diagnostics, basesBeingResolved);
```

Comment:
Constraint checks are not run on usings any more, they were never run on alias targets even before my change. They are verified indirectly when something gets bound through the using, that is why the error is reported on the use site of the using rather than on the declaration site.